docs/framework/interop/interop-marshaling.md

```diff
@@ -42,8 +42,6 @@ ms.author: "ronpet"
   
  At the lowest level, both mechanisms use the same interop marshaling service; however, certain data types are supported exclusively by COM interop or platform invoke. For details, see [Default Marshaling Behavior](default-marshaling-behavior.md).  
   
- [Back to top](#top)  
```

Comment:
With that PR, all `<a name="top"></a>` at the beginning of the files become useless.